project/SensibleSettings.scala

```diff
@@ -51,7 +51,7 @@ object Sensible {
     javaOptions += "-Dfile.encoding=UTF8",
     //this is needed by OrientDB versions 2.2.0+, 512g is recommended on their github wiki
     //https://github.com/orientechnologies/orientdb-docs/blob/master/Release-2.2.0.md
-    javaOptions += "-XX:MaxDirectMemorySize=512g",
+    javaOptions += "-XX:MaxDirectMemorySize=4g",
```

Comment:
I added this into the `EnsimeBuild.scala` in my branch. I think we should probably update the ensime 2.0 version of `sbt-ensime` to have this by default so that we don't have to tell people to do it.